backend/controllers/users.js

```diff
@@ -209,4 +209,73 @@ module.exports = {
       })
     );
   },
+  followUser: async (req, res) => {
```

Comment:
- create one function to toggle follow function. 
- inserting and deleting a row would be a good idea for this, since a user would probably click the button multiple times.
- instead insert a new column e.g. `flag` which is boolean. to determine if already followed. Set `1` for true and set `0` if unfollowed.